charts/incubator/zammad/values.yaml

```diff
@@ -0,0 +1,94 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: tccr.io/truecharts/zammad
+  tag: latest@sha256:c9c8ec07af85ca06ff2776e1bd00c70cca470c7bf9aa92c716b78283bf880f49
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+zammad:
+  zammad:
+    zammad_autowizardjson: ""
+  webconcurrency:
+    webcon_instances: ""
+    webcon_sessionjobs: 0
+    webcon_schedprocesses: false
+    webcon_delayedjobs: 0
+  rails:
+    rails_trustedproxies:
+    - 127.0.0.1
+    - ::1
+  elasticsearch:
+    elasticsearch_enabled: true
+    elasticsearch_port: 9200
+    elasticsearch_schema: "http"
+    elasticsearch_namespace: "zammad"
+    elasticsearch_reindex: true
+    elasticsearch_sslverify: true
```

Comment:
Why would those be exposed if the ES is used as a dependency? (meaning that no configuration is needed from the user)

Reply:
Good call, will fix this and update the documentation.